components/installer/cmd/operator/main.go

```diff
@@ -29,10 +29,14 @@ import (
 	informers "github.com/kyma-project/kyma/components/installer/pkg/client/informers/externalversions"
 )
 
+var gitCommitHash string
+
 func main() {
 
 	log.Println("Starting operator...")
 
+	log.Println("Git commit hash:", gitCommitHash)
```

Comment:
please check if gitCommitHash is not empty

Reply:
Done